portfolio/src/main/webapp/script.js

```diff
@@ -30,8 +30,10 @@ function addRandomGreeting() {
 async function fetchCall() {
   const response = await fetch('/hello'); 
 
-  const responseText = await response.text(); 
+  const responsejson = await response.json(); 
+  
+  var index = Math.floor(Math.random() * 3);  
 
   const textContainer = document.getElementById('text-container'); 
-  textContainer.innerText = responseText; 
+  textContainer.innerHTML = responsejson[Object.keys(responsejson)[index]]; 
```

Comment:
This is clever but why did you change `innerText` to `innerHTML`?

Reply:
I changed it because innerHTML returns the content in HTML format, the innerText returns a plain text.

Reply:
OK. I was being unclear. By assigning to `innerHTML`, you can have markup (and script) in the returned payload. But (to channel Spider Man) with great power comes great responsibility: the code that generates the string that you assign to an `innerHTML` block needs to always generate correct (and secure) HTML.

I don't think that you needed to make this change. There is no markup in the JSON payload and at some point, this could become a bug. It's also currently fine. In the future you should probably prefer to assign to `innerText` and use the DOM manipulation API.